src/Tribe/Editor.php

```diff
@@ -42,6 +42,45 @@ public function hook() {
 
 		// Make sure Events supports 'custom-fields'
 		add_action( 'init', array( $this, 'add_event_custom_field_support' ), 11 );
+
+		add_filter( 'use_block_editor_for_post_type', array( $this, 'deactivate_blocks_editor_venue' ), 10, 2 );
```

Comment:
Looks good. Let's add a todo so when `use_block_editor_for_{post_type}` is merged. 

- https://core.trac.wordpress.org/ticket/45275 

We can update this filter to use `__return_false` in the `callback` part instead.

Reply:
Ohh nice find.